apps/nextjs/src/middlewares/test/snapshots/policies.development.test.txt

```diff
@@ -0,0 +1,14 @@
+default-src 'self'
+media-src 'self' 'self' https://*.mux.com https://stream.mux.com blob:
```

Comment:
@codeincontext is this a bug? Should there be as string after blob:?

Reply:
Just a bug! Just an oddity

![CleanShot 2024-10-14 at 12 25 00@2x](https://github.com/user-attachments/assets/1f57f547-eec0-4954-afca-d18dc6d9630e)